### PR TITLE
Release 15.1.0

### PR DESCRIPTION
**Pull request by njuettner**

### CODE REVIEW

File: aws/v15.1.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v15.0.0                                                 |    name: v15.1.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - componentVersion: 1.1.3                                          - componentVersion: 1.1.3
+    name: aws-ebs-csi-driver                                           name: aws-ebs-csi-driver
+    version: 2.1.0                                                     version: 2.1.0
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.6.1                                                     version: 1.6.1
+  - componentVersion: 1.3.1                                     |    - name: cert-manager
```
Comment (alex-dabija) [inline aws/v15.1.0/release.diff:17]: `componentVersion` was removed. This is going to cause a UX issue in Happa.

File: aws/v15.1.0/release.diff [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    name: cert-manager                                          |      version: 2.8.0
+    version: 2.7.1                                              <
+  - name: chart-operator                                             - name: chart-operator
+    version: 2.14.0                                                    version: 2.14.0
+  - name: cluster-autoscaler                                         - name: cluster-autoscaler
+    version: 1.20.3                                                    version: 1.20.3
+  - componentVersion: 1.6.5                                     |    - name: coredns
```
Comment (alex-dabija) [inline aws/v15.1.0/release.diff:24]: `componentVersion` was removed. This is going to cause a UX issue in Happa.

File: aws/v15.1.0/release.diff [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    name: coredns                                               |      version: 1.6.9
+    version: 1.4.1                                              |    - name: external-dns
+  - componentVersion: 0.7.6                                     |      version: 2.4.0
```
Comment (alex-dabija) [inline aws/v15.1.0/release.diff:27]: `componentVersion` was removed. This is going to cause a UX issue in Happa.

File: aws/v15.1.0/release.diff [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    name: external-dns                                          |    - name: kiam
+    version: 2.3.1                                              <
+  - componentVersion: 3.6.0                                     <
+    name: kiam                                                  <
+    version: 1.7.1                                                     version: 1.7.1
+  - componentVersion: 1.9.7                                     |    - name: kube-state-metrics
```
Comment (alex-dabija) [inline aws/v15.1.0/release.diff:33]: `componentVersion` was removed. This is going to cause a UX issue in Happa.

File: aws/v15.1.0/release.diff [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    name: kube-state-metrics                                    |      version: 1.9.8
+    version: 1.3.1                                              |    - name: metrics-server
+  - componentVersion: 0.4.1                                     |      version: 0.4.4
```
Comment (alex-dabija) [inline aws/v15.1.0/release.diff:36]: `componentVersion` was removed. This is going to cause a UX issue in Happa.

File: aws/v15.1.0/release.diff [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    name: metrics-server                                        <
+    version: 1.3.0                                              <
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.10.1                                             |      version: 1.10.2
+  - componentVersion: 1.0.1                                     |    - name: node-exporter
```
Comment (alex-dabija) [inline aws/v15.1.0/release.diff:41]: `componentVersion` was removed. This is going to cause a UX issue in Happa.

File: aws/v15.0.0/release.yaml [hunk 7]
```diff
@@ -9,7 +9,7 @@ metadata:
   name: v15.0.0
 spec:
   apps:
-  - componentVersion: 1.1.0
+  - componentVersion: 1.1.3
```
Comment (alex-dabija) [inline aws/v15.0.0/release.yaml:12]: Why was the `componentVersion` changed for aws-ebs-cni-driver in v15.0.0? In the release diff of v15.1.0 `componentVersion` is set to 1.1.1. It looks to me like a mistake.

Comment (paurosello) [reply aws/v15.0.0/release.yaml:12]: This is what I saw in the changelog but could not find which is the one really there. @njuettner do you know?

Comment (njuettner) [reply aws/v15.0.0/release.yaml:12]: In release v15 we introduced aws-ebs-csi-driver, component version should be `1.1.0`

Comment (alex-dabija) [reply aws/v15.0.0/release.yaml]: I set the version to 1.1.0 for v15.0.0 .

**Review by alex-dabija — APPROVED** [review]
The release looks good from my side as long as the testing was successful (including in China).

**Pull request merged**